ui/security_tools_page.go

```diff
@@ -1,20 +1,28 @@
 package ui
 
 import (
+	"image"
+
 	"gioui.org/layout"
+	"gioui.org/widget"
+	"github.com/planetdecred/godcr/ui/values"
 
 	"github.com/planetdecred/godcr/ui/decredmaterial"
 )
 
 const PageSecurityTools = "Security_Tools"
 
 type securityToolsPage struct {
-	dummyText decredmaterial.Label
+	theme       *decredmaterial.Theme
+	verifyMess  *widget.Clickable
```

Comment:
You can make these variable names complete "verifyMessage"  instead of "verifyMess", same thing with "validateAdd"